micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductDetails/ProductDetailsComponentUpdated.js

```
@@ -0,0 +1,349 @@
+        <div className="role-block">
+            <div className="role-icon-wrapper">
+                {isCitizen ? <Citizenicon /> : <Employeeicon />}
+            </div>
+            <p dangerouslySetInnerHTML={{ __html: decodedDescription }} />
+        </div>
+    );
+};
+
+
+
+const ExperienceSection = ({ experience, t = { t } }) => (
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)